opm-core/Installation.rst

```diff
@@ -157,35 +153,29 @@ The dispatcher itself::
 
 .. note::
 
-    With our previous examples, **db_user** would've been set to ``user1`` and
-    **db_password** should be set to ``password1``.
+    With our previous examples, **db_user** would've been set to ``opm`` and
+    **db_password** should be set to ``anothergoodpassword``. If you use Icinga2,
+    ``directory`` must be set to ``/var/spool/icinga2/perfdata/``.
 
-Install the nagios_dispatcher.pl file into the /usr/local/bin/ directory::
+Install the ``nagios_dispatcher.pl`` file into ``/usr/local/bin/`` ::
 
     root:~# cp /usr/local/src/opm/wh_nagios/bin/nagios_dispatcher.pl /usr/local/bin
 
+You can test that it works on the command line (you may have to set ``daemon=0``)::
+
+    /usr/local/bin/nagios_dispatcher.pl --verbose -c /usr/local/etc/nagios_dispatcher.conf
+
+The files in the Nagios ``perfdata`` directory must disappear one by one.
+
+
 **If your operating system uses systemd**
 
-Slight change to the nagios_dispatcher.cfg file::
+Modify the ``nagios_dispatcher.conf`` above like this::
```

Comment:
It'd be better to document that:

  * daemon parameter should be changed to 0 intead of 1
  * and still give the hole content, so user can easily copy/paste or double check everything is ok

Reply:
I've added the file back.